src/frontend/src/lib/components/onramper/OnramperWidget.svelte

```diff
@@ -0,0 +1,68 @@
+<script lang="ts">
+	import { nonNullish } from '@dfinity/utils';
+	import { NETWORK_BITCOIN_ENABLED } from '$env/networks.btc.env';
+	import { BTC_MAINNET_TOKEN } from '$env/tokens.btc.env';
+	import { ICP_TOKEN } from '$env/tokens.env';
+	import { ethereumToken } from '$eth/derived/token.derived';
+	import { icrcAccountIdentifierText } from '$icp/derived/ic.derived';
+	import { btcAddressMainnet, ethAddress } from '$lib/derived/address.derived';
+	import { networkBitcoin, networkEthereum } from '$lib/derived/network.derived';
+	import { networks } from '$lib/derived/networks.derived';
+	import { tokens } from '$lib/derived/tokens.derived';
+	import { token } from '$lib/stores/token.store';
+	import type { OnramperCryptoWallet, OnramperId, OnramperNetworkId } from '$lib/types/onramper';
+	import { buildOnramperLink, mapOnramperWallets } from '$lib/utils/onramper.utils';
+
+	let defaultCrypto: OnramperId | undefined;
+	$: defaultCrypto =
+		$token?.buy?.onramperId ??
```

Comment:
Shouldn't we pass the token(s) as property(-ies) or it's always global?

Reply:
I didn't think necessary to pass it, since those configs are quite global. I was even thinking of creating specific derived stores for it, but then it's too much

Reply:
Yeah too much stores kill the stores. Thanks for the feedback, global it is.